src/Microsoft.ML.FastTree/TreeEnsembleFeaturizer.cs

```diff
@@ -387,7 +387,7 @@ public TreeEnsembleFeaturizerBindableMapper(IHostEnvironment env, ModelLoadConte
             _totalLeafCount = CountLeaves(_ensemble);
         }
 
-        public void Save(ModelSaveContext ctx)
+        void ICanSaveModel.Save(ModelSaveContext ctx)
```

Comment:
>void ICanSaveModel.Save [](start = 8, length = 23)

Note that even though this is itself an internal class, I still appreciate making these implementations explicit, since I help it helps establish the pattern. (So I like this change, in case not clear. :) )(

Reply:
Ok makes sense :)

---
In reply to: [254472337](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254472337) [](ancestors = 254472337)